packages/roosterjs-react-command-bar/lib/utils/OutOfBoxCommandBarButtons.tsx

```diff
@@ -118,7 +118,6 @@ export const OutOfBoxCommandBarButtons: RoosterCommandBarButtonInternal[] = [
         },
         subMenuProps: {
             className: "rooster-command-bar-color-container",
-            key: "highlight-sub",
```

Comment:
Whats the reason for removing the key here?

Reply:
This property does nothing in OUFR. subMenuProps didn't need distinct keys because the submenu doesn't appear in a list -- the button does, and those already have keys defined.